imagetool/src/main/java/com/oracle/weblogic/imagetool/cli/menu/WdtOptions.java

```diff
@@ -53,12 +50,7 @@
             dockerfileOptions.setWdtModels(modelList);
 
             dockerfileOptions.setWdtDomainType(wdtDomainType);
-            if (wdtDomainType != DomainType.WLS) {
-                if (installerType != FmwInstallerType.FMW) {
-                    throw new IOException("FMW installer is required for JRF domain");
-                }
-                dockerfileOptions.setRunRcu(runRcu);
-            }
+            dockerfileOptions.setRunRcu(runRcu);
```

Comment:
Will this be a problem with non FMW typed domain and run_rcu set?

Reply:
Why would it?  This particular line is passing the Image Tool command line option onto WDT.  If the option is not provided on the Image Tool command line, the default is false (same as WDT).

Reply:
That should be ok, if user pass in the flag for a non FMW domain then it should fail as expected.